topostats/grains.py

```diff
@@ -2443,3 +2379,72 @@ def remove_disconnected_grains(
                     predicted_grain_tensor[predicted_mask_region_mask, channel] = 0
 
         return predicted_grain_tensor
+
+
+# pylint: disable=too-many-branches
+def get_grain_thresholds(  # noqa: C901
+    image: npt.NDArray,
+    threshold_method: str,
+    otsu_threshold_multiplier: float | None = None,
+    threshold_std_dev: list | None = None,
+    absolute: list | None = None,
```

Comment:
Consistent suffix


```suggestion
    threshold_absolute: list | None = None,
```

Use IDE to replace throughout.